### PR TITLE
Fix/build gazebo8

### PR DESCRIPTION
**Pull request by devbharat**
Fix build issues when using gazebo8.

(We need to push the rotor_simulation integration harder to not have repeated fixes.)

### CODE REVIEW

File: src/gazebo_gimbal_controller_plugin.cpp [hunk 1]
```diff
@@ -308,9 +308,13 @@ void GimbalControllerPlugin::OnUpdate()
     /// and gimbal is constructed using yaw-roll-pitch-variable-axis
     ignition::math::Vector3d currentAngleYPRVariable(
       this->imuSensor->Orientation().Euler());
+#if GAZEBO_MAJOR_VERSION >= 8
```
Comment (pickledgator) [inline src/gazebo_gimbal_controller_plugin.cpp:311]: You don't actually have to wrap this in an #if. This code will also work with the older versions which used ign_math2 and didn't have an explicit tag on the quaternion constructor.

**Comment by pickledgator** [comment]
Besides these changes, have you gotten sitl_gazebo fully running on gazebo8? Did you have to make any other modifications? I had some runtime errors that were difficult to diagnose (running sdformat 5.1.0, transport 3.0.1, msgs 0.7.0 and math 3.2.0.)

File: src/gazebo_gimbal_controller_plugin.cpp (hunk 1 above)
Comment (kgreenek) [reply src/gazebo_gimbal_controller_plugin.cpp:311]: This should work:
```c++
ignition::math::Quaternion currentAngleYPRVariable(
    this->imuSensor->Orientation().Euler()); 
````

**Comment by LorenzMeier** [comment]
Thanks! I pulled this indirectly in.

**Pull request closed without merging**